Dfe.Data.SearchPrototype/SearchForEstablishments/Models/FacetResult.cs

```diff
@@ -0,0 +1,28 @@
+﻿namespace Dfe.Data.SearchPrototype.SearchForEstablishments.Models;
+
+/// <summary>
+/// The object that encapsulates a single facet result for a facet 
+/// </summary>
+public class FacetResult
+{
+    /// <summary>
+    /// The value of the facet result
+    /// </summary>
+    public string Value { get; }
+
+    /// <summary>
+    /// The number of records that belong to this facet value
+    /// </summary>
+    public int? Count { get; }
+
+    /// <summary>
+    /// Constructor
+    /// </summary>
+    /// <param name="value">The value of the facet result</param>
+    /// <param name="count">The number of records that belong to this facet value</param>
+    public FacetResult(string value, int? count)
+    {
+        Value = value;
+        Count = count;
+    }
+}
```

Comment:
Personally I would tend to use records for types like this, but appreciate this kind of comment is bigger than just this PR.

Reply:
yes, agreed. When we started the project we were going to use records but because we were rusty on them and were all wanting to start we went with classes. We intend to move to records where appropriate and should do so asap because the longer we leave it the more work there is to do to change to using records.